### DISCUSSION TITLE
UX. E eu com isso?

### DISCUSSION THREAD

**Discussion by luisfmsouza** [frontendbr/forum#73] [Discussão]
- Como funciona no projeto de vocês a integração das equipes de front-end e UX(se tiver)?
- Até onde vocês acham importante a participação do front-end nas decisões do time de UX?
- Vocês estudam/leem sobre o assunto?

**Comment by felipefialho** [comment]
@luisfmsouza UX é um dos assuntos que mais leio diariamente. 

Considero essencial que a gente tenha um bom entendimento do assunto e muito importante que a gente participe de decisões nesse sentido (mesmo não senda da nossa total responsabilidade).

**Comment by vitor-mendrone-deel** [comment]
Concordo com o @LFeh 

Diariamente tenho lido e estudado mais sobre o assunto por entender que é vital ao desenvolvedor frontend ter plena compreensão do assunto. 

Cabe a nós garantir que a interface da aplicação/site funcione corretamente, assim como parte do trabalho de performance que jamais pode ser ignorada quando se trabalha com foco em UX. 

Claro que a UX vai muito além de nossas responsabilidades, designers, backenders, marketing, suporte, etc, todos tem participação nesse processo, já que ter ou não uma boa experiência com um produto vai além do produto em si.

**Comment by luisfmsouza** [comment]
Concordo plenamente com vocês. 

Comecei a trabalhar com um time de UX recentemente e já mudei bastante a maneira como eu encaro as coisas, agora sempre tenho isso em mente quando vou pensar nas soluções.

Vocês comentaram que leem bastante sobre o assunto, compartilhem as fontes ai @LFeh e @Mendrone

**Comment by brunoravanhani** [comment]
Na empresa onde trabalho infelizmente não há uma equipe de UX, nem uma preocupação com isso, UX aqui é o que o meu chefe acha que é melhor rsrsrs... Mas como se eu não vejo o cara do UX da empresa, então eu sou o cara da UX da empresa, vou tentando convencer a chefia em algumas coisas.

Mas eu acredito ser essencial para o desenvolvedor Front-End. É o assunto que mais eu leio também. 

As minhas principais fontes de conteúdo são: o [blog de ai](http://arquiteturadeinformacao.com/), no [medium](https://medium.com/tag/ux) tem muita coisa legal e também o [Webs Designer News](http://www.webdesignernews.com/) que lota a minha caixa de entrada, mas é muito bom.

**Comment by felipefialho** [comment]
@luisfmsouza Eu costumo compartilhar TUDO o que eu leio no Twitter, basta me seguir que vai ter informações o tempo todo, haha.

Mas tenho um feed-list marota com algumas coisas em português - https://github.com/LFeh/feed-list

Fora isso costumo olhar no Medium.

**Comment by bernardodiasc** [comment]
Vi esse video recentemente, bem interessante http://99percentinvisible.org/article/norman-doors-dont-know-whether-push-pull-blame-design/

**Comment by bukinoshita** [comment]
Opa, eu sou UX Lead Designer & Front-end Developer lá em Toronto.

## Até onde vocês acham importante a participação do front-end nas decisões do time de UX?

Todo tempo, mas tem um porém. Como UX é uma parada que ainda está crescendo e muitas empresas ainda não levam o design a sério ou acham que levam, é algo complicado.

Como UX Lead Designer, eu tenho que ficar o tempo todo ensinando as pessoas, a própria empresa (developers/marketing/PM/CEO) a entenderem o processo de UX, o motivo das decisões tomadas, o porque colocar ou tirar tal coisa, todo o processo de design. Mas como você sabem não da pra fazer tudo do seu jeito, tem que seguir um lado business também. Algo que seja tangível tanto pro lado do business e o lado de UX.

**TLDR: A gente tem o problema, resolve o problema do lado do business e depois resolve o lado do UX (duas diferentes versões de produto), depois molda que encaixe dos dois lado, ai você tem o produto.**

(acaba respondendo a primeira questão, não sei se expliquei bem, mas se tiverem dúvida podem perguntar).

## Vocês estudam/leem sobre o assunto?

O tempo todo.

**Comment by rubenmarcus** [comment]
o que acham do layout do G1?

E de portfolios feito em one scroll page?

Citem ai bons projetos de UX que voces gostam ou participaram

vou citar um que eu gosto : Mashable

**Comment by luisfmsouza** [comment]
@BuKinoshita bacana demais sua resposta, pra vermos o contraponto da visão dev.

Concordo que os profissionais de UXs, além de fazer o trabalho do dia-a-dia, também precisam evangelizar as pessoas/empresas sobre a importância do próprio UX. É um trabalho duro! Mas quando todos entendem, e UX entra de fato no processo de desenvolvimento, os resultados são nítidos.

**Comment by felipefialho** [comment]
@rubenmarcus 

#### Sobre o G1

Não estou exagerando: Acho que o G1 tem o melhor layout entre os sites de pesquisas DO MUNDO. É bem acima dos demais.

#### Onepage Scroll

Curto só para alguns projetos pontuais, para portfólio, depende do objetivo, pode ficar legal, ou ficar uma bomba. Não tem meio termo.

**Comment by rubenmarcus** [comment]
@LFeh  então voce é a favor de sites como o g1, o terra e etc? o que eu vi de usuário reclamando desses sites...

**Comment by felipefialho** [comment]
O Terra não curto.

Já o G1 acho realmente muito bom.

**Comment by tcelestino** [comment]
Sempre acreditei que o profissional front end, principalmente que trabalha com UX, tem que está estudando sobre o assunto. 

Existem diversos blogs e muitos artigos no Medium que mostra bastante o que está acontecendo no mundo do UX. Através do @LFeh, conheci o blog [Arquitetura da Informação](http://arquiteturadeinformacao.com/) que tem conteudo pra caramba.

Aqui no R7, recentemente a equipe de front end começou ter mais ligação a equipe de UX e Design, e o trabalho está sendo feito sem menos "vai-e-volta".

**Comment by bukinoshita** [comment]
Um ponto que quero levantar aqui. Front-end Developers acham que sabem UX, acham que entendem UX só por estarem sempre lendo sobre UX em blogs etc... Isso ajuda? Ajuda, mas não é UX, é intuição. O "UX" não vale nada se não tiver pesquisas e testes sobre seus usuários.

**Comment by felipefialho** [comment]
Tem razão @BuKinoshita, eu leio, tento entender. Mas nem de longe poderia ter uma responsabilidade dessa como 'minha' nos projetos. 

Só dou meus pitacos por causa dos anos de experiencia e das leituras ;)

**Comment by lagden** [comment]
UX é very easy!!

Eu chego pra minha mãe e digo assim: `Mãe, compra um bolsa nessa loja virtual!`
Se a bolsa chegar UX aprovado!!
Hihihi... :metal:

**Comment by wesleyalmd** [comment]
@lagden Já vem mensurando seus estudos de UX em campo! rsrs

@BuKinoshita você como um profissional focado na área, conseguiria dar diretrizes para quem tem interesse em avançar na área? Usando eu mesmo como exemplo, sou desenvolvedor frontend, utilizo boas práticas e estudo bastante o assunto, mais como resultado final quero ter certeza das minhas tomadas de decisão e me qualificar muito! rs

**Comment by lagden** [comment]
@wesleyalmd foi uma brincadeira, mas aqui as vezes rola uns testes com as mamães mesmo.
UI/UX é a parte treta do Frontend.

Imagem clássica!!
![ux](https://s-media-cache-ak0.pinimg.com/564x/16/d9/8a/16d98abf2386e2fb5974e2d360250639.jpg)

**Comment by luisfmsouza** [comment]
Boa ideia @wesleyalmd !

@BuKinoshita ajude-nos a sermos melhores em UX :)

**Comment by bukinoshita** [comment]
Opa @wesleyalmd @luisfmsouza , vou escrever um post sobre UX quando tiver mais tempo e posto aqui. To um pouco corrido com as coisas, nem deu pra responder hahah

Mas basicamente UX -> Research / User Research.
